tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>How is this class meant to be used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>As a pure mixin. Added some extra explanation, here and for <code>MyFluid</code>.</p>